### PR TITLE
Update DPA3-v2 to DPA-3.1

### PR DESCRIPTION
**Pull request by iProzd**
## Description

Dear Devs,

Thank you for your continued efforts in maintaining the Matbench leaderboard.

We would like to request an update to our previous submission by replacing the DPA3-v2 model with the improved DPA-3.1 version. Additionally, we have an updated link to the corresponding paper.

Please let us know if any further adjustments are needed. We appreciate your guidance in ensuring our submission aligns with the repository’s standards.

Best regards,
Duo

## Checklist

Please check the following items before submitting your PR:

- [x] I created a new folder and YAML metadata file `models/<arch_name>/<model_variant>.yml` for my submission. `arch_name` is the name of the architecture and `model_variant.yml` includes things like author details, training set names and important hyperparameters.
- [x] I added the my new model as a new attribute on the [`Model.<arch_name>` enum](https://github.com/janosh/matbench-discovery/blob/57d0d0c8a14cd317/matbench_discovery/enums.py#L274) in `enums.py`.
- [x] I uploaded the energy/force/stress model prediction file for the WBM test set to Figshare or another cloud storage service (`<yyyy-mm-dd>-<model_variant>-preds.csv.gz`).
- [x] I uploaded the model-relaxed structures file to Figshare or another cloud storage service in [JSON lines format](https://jsonlines.org) (`<yyyy-mm-dd>-wbm-IS2RE-FIRE.jsonl.gz`). JSON Lines allows fast loading of small numbers of structures with `pandas.read_json(lines=True, nrows=100)` for inspection.
- [x] I uploaded the phonon predictions to Figshare or another cloud storage service (`<yyyy-mm-dd>-kappa-103-FIRE-<values-of-dist|fmax|symprec>.gz`).
- [x] I included the urls to the Figshare files in the YAML metadata file (`models/<arch_name>/<model_variant>.yml`). If not using Figshare I have included the urls to the cloud storage service in the description of the PR.
- [x] I included the test script (`test_<arch_name>_<task>.py` for `task` in `discovery`, `kappa`, `diatomics`) that generated the prediction files.

## Additional Information (Optional)

- [ ] I included a training script (`train_<arch_name>.py`) if I trained a model specifically for this benchmark.
- [x] I included a `readme.md` with additional details about my model.

### CODE REVIEW

**Review by janosh — COMMENTED** [review]
nice work on this model submission!

in `models/deepmd/dpa-3.1-mptrj.yml`, `pred_file: models/deepmd/dpa-3.1-3M-ft/2025-06-05-wbm-IS2RE-FIRE.jsonl.gz` is the same as in `models/deepmd/dpa-3.1-3m-ft.yml`

**Comment by iProzd** [comment]
> nice work on this model submission!
> 
> in `models/deepmd/dpa-3.1-mptrj.yml`, `pred_file: models/deepmd/dpa-3.1-3M-ft/2025-06-05-wbm-IS2RE-FIRE.jsonl.gz` is the same as in `models/deepmd/dpa-3.1-3m-ft.yml`

@janosh Thank you for catching this! I've re-uploaded both files and verified each model now references its prediction file. Please let me know if there's any other issues.

**Review by janosh — APPROVED** [review]
excellent! all looks good now. congrats to the whole DPA3 team on this achievement 👍
